iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -1,5 +1,6 @@
 package com.linkedin.openhouse.internal.catalog;
 
+import static com.linkedin.openhouse.internal.catalog.CatalogConstants.*;
```

Comment:
lets remove * and declare explicit constants?

Reply:
addressed.